### PR TITLE
Finished the endpoints-json file with all of the endpoints and example responses for each one 

### PR DESCRIPTION
**Pull request by Nosajthe1**
Completed endpoints.json file with the following endpoints and their respective responses -
 - GET /       - serves a .JSON representation of all the available endpoints of the API.
 - GET /sites      - serves an array of all sites.
 - POST /sites        - Adds a new site into the sites database.
 - GET /sites/:site_id       - Provides an object of a single site with specified site id.
 - PATCH /sites/:site_id     - Changes the specified property/properties of a specified site by a number/string which is passed in through the request
 - DELETE /sites/:site_id       - Deletes the specified site and responds with a status 204
 - GET /tours    - serves an array of all tours
 - POST /tours       - Adds a new tour into the tours database
 - GET /tours/:tour_id         - provides an object of a single tour with specified tour id
 - PATCH /tours/:tour_id          - Changes the specified property/properties of a specified tour by a number/string which is passed in through the request
 - DELETE /tours/:tour_id           - Deletes the specified tour and responds with a status 204

### CODE REVIEW

File: endpoints.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          "websiteLink": "https://www.durhamcathedral.co.uk/"
+        }
+      ]
+    }
+  },
+  "POST /sites": {
+    "description": "Adds a new site into the sites database",
+    "queries": [],
+    "exampleRequest": {
+      "siteId": 27,
```
Comment (BrennB99) [inline endpoints.json:29]: In the post requests the siteId/tourId is automatically created so shouldn't be in the request body

File: endpoints.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        }
+      ]
+    }
+  },
+
+  "POST /tours": {
+    "description": "Adds a new tour into the tours database",
+    "queries": [],
+    "exampleRequest": {
+      "tourId": 13,
```
Comment (BrennB99) [inline endpoints.json:116]: same as above

**Review by BrennB99 — APPROVED** [review]
Once the Id's are removed it's ready to merge

**Review by callumu21 — APPROVED** [review]
Looks good to me!

**Pull request merged**